### PR TITLE
C24_WPDE_Desktop_01

### PR DESCRIPTION
**Pull request by gbirke**
Ticket: https://phabricator.wikimedia.org/T375609

### CODE REVIEW

**Comment by Sperling-0** [comment]
⚠️ Rebase this PR from latest main.

To get the latest UoF working:

- For Desktop banners: `BannerCtrl.vue`, `BannerVar.vue`, `FallbackBanner.vue` needs to be updated. (If it is DE desktop include DE infographic and vice versa)
- For Mobile/ Ipad banners: `BannerCtrl.vue`, `BannerVar.vue` needs to be updated. (If it is DE mobile include DE infographic and vice versa)

**Comment by moiikana** [comment]
>[!TIP]
> rebasing might be relevant here because  https://github.com/wmde/fundraising-banners/pull/590 got merged

https://github.com/wmde/fundraising-banners/pull/585 as well

File: banners/wpde_desktop/C24_WPDE_Desktop_01/components/BannerCtrl.vue [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+		</FundsModal>
+
+		<SoftClose
+			v-if="contentState === ContentStates.SoftClosing"
+			@close="() => onClose( 'SoftClose', CloseChoices.Close )"
+			@maybe-later="() => onClose( 'SoftClose', CloseChoices.MaybeLater )"
+			@time-out-close="() => onClose( 'SoftClose', CloseChoices.TimeOut )"
+		/>
+
+		<AlreadyDonatedModal
```
Comment (Abban) [inline banners/wpde_desktop/C24_WPDE_Desktop_01/components/BannerCtrl.vue]: This is removed in the other banners, so we might ask the PM if it's to also be removed here

**Review by Abban — APPROVED** [review]
We might try adding a little more padding to the left of the radios, the text is very close:

Screenshot 2024-10-24 at 14 33 54

**Comment by gbirke** [comment]
> We might try adding a little more padding to the left of the radios, the text is very close:
> 
> Screenshot 2024-10-24 at 14 33 54

This is all configured in the `Treedip` theme. I don't want to change it there to avoid affecting the English Desktop banner that uses that theme as well. I diffed the "calculated styles" from the inspector for EN and WPDE and they are functionally the same (WPDE has a few more font settings). Of course I can add some custom style to the banner itself, but I'd rather get to the bottom of this. If you have ideas on how to debug this or what could be causing this, I'd be very grateful.

**Comment by Abban** [comment]
There's something different happening in the EN banner though, as the radios are a little further to the left in it. Let me investigate.

File: banners/wpde_desktop/C24_WPDE_Desktop_01/styles/styles.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+@use 'Banner';
+@use 'MainBanner' with (
+	$banner-height: 394px,
+	$form-width: 300px
+);
+@use '../../../../src/themes/UseOfFunds/swatches/skin_default' as uof-default;
+@use '@src/themes/UseOfFunds/UseOfFunds';
+@use '@src/themes/Treedip/defaults';
+@use '@src/themes/Treedip/ButtonClose/ButtonClose';
+@use '@src/themes/Treedip/ProgressBar/ProgressBar' with (
```
Comment (Abban) [inline banners/wpde_desktop/C24_WPDE_Desktop_01/styles/styles.scss:23]: On line 58 of `ProgressBar.scss` there's a line height rule (`line-height: 26px;`) that you can delete. It's breaking the alignment of the inner text. It won't affect the EN banner.

**Comment by Abban** [comment]
Found the issue with the radios. In `Treedip/DonationForm/BubbleForm/radio-input.scss` the mixin margin should be:

`margin: -( math.div( $size, 2 ) ) 10px 0 0;`

**Comment by moiikana** [comment]
consider rebasing because https://github.com/wmde/fundraising-banners/pull/597 got merged

**Review by moiikana — APPROVED** [review]

**Pull request merged**